lib/screens/forgot_password/forgot_password_repository.dart

```diff
@@ -0,0 +1,43 @@
+import 'dart:math';
+
+import 'package:ecellapp/core/res/errors.dart';
+
+abstract class ForgotPasswordRepository {
+  Future<void> sendOTP(String email);
+
+  Future<bool> verifyOTP(String otp);
+
+  Future<void> changePassword(String email, String otp, String password);
+}
+
+// To emulate a fake process for forgot password
+class FakeForgotPasswordRepository implements ForgotPasswordRepository {
```

Comment:
extend it instead of implementing.
```suggestion
class FakeForgotPasswordRepository extends ForgotPasswordRepository {
```